scripts/process.py

```diff
@@ -1,3 +1,9 @@
+'''
+This script will import and clean data from the JHU CSSE COVID-19 dataset.
```

Comment:
This is true but not necessary here. The README has more information, provides a list of sources and should be read first, hence the name `README`.